README.md

```diff
@@ -8,35 +8,36 @@ This repo provides both a library for zingoproxyclient and zingo-mobile, as well
+* This software does not provide any privacy guarantees against network monitoring of the type or pattern of traffic it generates. For example, a user's specific activities using the client may remain private, but the use of the tool itself may be apparent to network observers.
 
 ## Zingo CLI
 `zingo-cli` is a command line Zingo lightwalletd-proxy client. To use it, see "compiling from source" below. Releases are currently only provisional, we will update the README as releases come out.
 
 ## Privacy 
-* While all the keys and transaction detection happens on the client, the server can learn what blocks contain your shielded transactions.
+* While all the keys are stored locally, and transaction detection happens in the client, the server can learn what blocks contain your shielded transactions.
 * The server also learns other metadata about you like your ip address etc...
-* Also remember that t-addresses don't provide any privacy protection.
+* Also remember that t-addresses are not private on the blockchain.
 
 ### Note Management
 Zingo-CLI does automatic note and utxo management, which means it doesn't allow you to manually select which address to send outgoing transactions from. It follows these principles:
-* Defaults to sending shielded transactions, even if you're sending to a transparent address
-* Sapling funds need at least 5 confirmations before they can be spent
-* Can select funds from multiple shielded addresses in the same transaction
-* Will automatically shield your transparent funds at the first opportunity
-    * When sending an outgoing transaction to a shielded address, Zingo-CLI can decide to use the transaction to additionally shield your transparent funds (i.e., send your transparent funds to your own shielded address in the same transaction)
+* Defaults to sending shielded transactions, even if you're sending to a transparent address.
+* Sapling funds need at least 5 confirmations before they can be spent.
+* Can use funds from multiple shielded addresses in the same transaction.
+* Will automatically shield your transparent funds at the first opportunity: when sending an outgoing transaction to any shielded address, Zingo-CLI will use the transaction to additionally shield your transparent funds. That is, it will send your transparent funds to your own shielded address in the same transaction. The `shield` command performs a similar function, but without sending to a shielded address outside of your wallet.
 
 ## Compiling from source
 
 #### Pre-requisites
 * Rust v1.37 or higher.
-    * Run `rustup update` to get the latest version of Rust if you already have it installed
+    * Run `rustup update` to get the latest version of Rust if you already have it installed.
```

Comment:
This is beyond our scope.   We should just specify the version of Rust we are using, and let the user make their own decisions about updating their toolchain.

Reply:
This was in the inherited README.

Reply:
OK.